tests/by-util/test_chgrp.rs

```diff
@@ -1,5 +1,6 @@
 use crate::common::util::*;
 use rust_users::*;
+use uucore;
```

Comment:
please fix this warning

Reply:
I am not sure why this is claimed as unused because I definitely use it down below, but I can import the submodule and see fi that makes the warning go away